2022-06-18T12:22:14.723Z - info: [ui] [AppStore] loading 1 repositories from store
2022-06-18T12:22:14.731Z - info: [ui] [AppStore] found account: xandernp (Xander Newman-Peill)
2022-06-18T12:22:14.840Z - info: [ui] launching: 3.0.0-linux2 (Linux 5.18.5-arch1-1)
2022-06-18T12:22:14.840Z - info: [ui] execPath: '/opt/github-desktop/github-desktop'
2022-06-18T12:22:15.078Z - info: [ui] Subscribed 'xandernp' to Alive channel
2022-06-18T12:22:15.169Z - info: [ui] Stats reported.
2022-06-18T12:24:34.017Z - info: [ui] [AppStore] loading 0 repositories from store
2022-06-18T12:24:34.023Z - info: [ui] [AppStore] found account: xandernp (Xander Newman-Peill)
2022-06-18T12:24:34.114Z - info: [ui] launching: 3.0.0-linux2 (Linux 5.18.5-arch1-1)
2022-06-18T12:24:34.116Z - info: [ui] execPath: '/opt/github-desktop/github-desktop'
2022-06-18T12:24:34.473Z - info: [ui] Subscribed 'xandernp' to Alive channel
2022-06-18T12:24:44.228Z - info: [ui] [AppStore.getAccountForRemoteURL] account found for remote: https://github.com/xandernp/config.git - xandernp (has token)
2022-06-18T12:26:03.756Z - info: [ui] Executing clone: git -c credential.helper= -c protocol.version=2 -c init.defaultBranch=main clone --recursive --progress -- https://github.com/xandernp/config.git /home/xandernp/GitHub/config (took 79.486s)
2022-06-18T12:26:03.779Z - info: [ui] [AppStore] adding repository at /home/xandernp/GitHub/config to store
2022-06-18T21:08:45.301Z - info: [ui] [AppStore] loading 1 repositories from store
2022-06-18T21:08:45.306Z - info: [ui] [AppStore] found account: xandernp (Xander Newman-Peill)
2022-06-18T21:08:45.357Z - info: [ui] [BranchPruner] Last prune took place 9 hours ago - skipping
2022-06-18T21:08:45.457Z - info: [ui] launching: 3.0.0-linux2 (Linux 5.18.5-arch1-1)
2022-06-18T21:08:45.457Z - info: [ui] execPath: '/opt/github-desktop/github-desktop'
2022-06-18T21:08:45.698Z - info: [ui] Subscribed 'xandernp' to Alive channel
2022-06-18T21:08:46.961Z - info: [ui] Executing getStatus: git --no-optional-locks status --untracked-files=all --branch --porcelain=2 -z (took 1.512s)
2022-06-18T21:08:47.072Z - info: [ui] Executing getStatus: git --no-optional-locks status --untracked-files=all --branch --porcelain=2 -z (took 1.429s)
2022-06-18T21:08:47.356Z - info: [ui] Executing getStatus: git --no-optional-locks status --untracked-files=all --branch --porcelain=2 -z (took 2.041s)
2022-06-18T21:08:47.692Z - info: [ui] Executing getStatus: git --no-optional-locks status --untracked-files=all --branch --porcelain=2 -z (took 2.079s)
2022-06-18T21:08:52.779Z - info: [ui] Executing unstageAll: git reset -- . (took 1.310s)
2022-06-18T21:08:53.080Z - info: [ui] [Timing] Action 'create commit' for 'xandernp/.config' took 1.644s
2022-06-18T21:08:53.794Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: .config - xandernp (has token)
2022-06-18T21:08:54.411Z - error: [ui] `git -c credential.helper= -c protocol.version=2 push origin main:main --progress` exited with an unexpected code: 1.
stderr:
To https://github.com/xandernp/.config.git
 ! [rejected]        main -> main (fetch first)
error: failed to push some refs to 'https://github.com/xandernp/.config.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

(The error was parsed as 10: The repository has been updated since you last pulled. Try pulling before pushing.)
2022-06-18T21:08:57.937Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: .config - xandernp (has token)
2022-06-18T21:09:00.364Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: .config - xandernp (has token)
2022-06-18T21:09:00.940Z - error: [ui] `git -c credential.helper= -c protocol.version=2 -c rebase.backend=merge pull --ff --progress origin` exited with an unexpected code: 1.
stdout:
Auto-merging README.md
CONFLICT (content): Merge conflict in README.md
Automatic merge failed; fix conflicts and then commit the result.

(The error was parsed as 7: We found some conflicts while trying to merge. Please resolve the conflicts and commit the changes.)
2022-06-18T21:09:06.701Z - error: [ui] Highlighter worked failed for new contents
Error: timed out
    at /home/xandernp/webpack:/[name]/app/src/lib/highlighter/worker.ts:85:14
2022-06-18T21:09:12.450Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: .config - xandernp (has token)
2022-06-18T21:09:12.538Z - error: [ui] `git -c credential.helper= -c protocol.version=2 -c rebase.backend=merge pull --ff --progress origin` exited with an unexpected code: 128.
stderr:
error: Pulling is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.

(The error was parsed as 38: There are unresolved conflicts in the working directory.)
2022-06-18T21:09:18.318Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: .config - xandernp (has token)
2022-06-18T21:09:18.958Z - error: [ui] `git -c credential.helper= -c protocol.version=2 -c rebase.backend=merge pull --ff --progress origin` exited with an unexpected code: 1.
stdout:
Auto-merging README.md
CONFLICT (content): Merge conflict in README.md
Automatic merge failed; fix conflicts and then commit the result.

(The error was parsed as 7: We found some conflicts while trying to merge. Please resolve the conflicts and commit the changes.)
2022-06-18T21:09:24.171Z - error: [ui] Highlighter worked failed for new contents
Error: timed out
    at /home/xandernp/webpack:/[name]/app/src/lib/highlighter/worker.ts:85:14
2022-06-18T21:09:32.084Z - info: [ui] [AppStore] loading 1 repositories from store
2022-06-18T21:09:32.088Z - info: [ui] [AppStore] found account: xandernp (Xander Newman-Peill)
2022-06-18T21:09:32.108Z - info: [ui] Subscribed 'xandernp' to Alive channel
2022-06-18T21:09:32.132Z - info: [ui] [BranchPruner] Last prune took place 1 minute ago - skipping
2022-06-18T21:09:32.496Z - info: [ui] launching: 3.0.0-linux2 (Linux 5.18.5-arch1-1)
2022-06-18T21:09:32.497Z - info: [ui] execPath: '/opt/github-desktop/github-desktop'
2022-06-18T21:09:37.873Z - error: [ui] Highlighter worked failed for new contents
Error: timed out
    at /home/xandernp/webpack:/[name]/app/src/lib/highlighter/worker.ts:85:14
2022-06-18T21:09:47.031Z - info: [ui] [AppStore.getAccountForRemoteURL] account found for remote: https://github.com/xandernp/.config.git - xandernp (has token)
2022-06-18T21:11:58.947Z - info: [ui] Executing clone: git -c credential.helper= -c protocol.version=2 -c init.defaultBranch=main clone --recursive --progress -- https://github.com/xandernp/.config.git /home/xandernp/GitHub/.config (took 131.855s)
2022-06-18T21:11:58.966Z - info: [ui] [AppStore] adding repository at /home/xandernp/GitHub/.config to store
2022-06-18T21:11:59.395Z - info: [ui] [BranchPruner] Last prune took place 3 minutes ago - skipping
2022-06-18T21:11:59.420Z - info: [ui] [BranchPruner] Last prune took place 3 minutes ago - skipping
2022-06-18T21:12:00.692Z - info: [ui] Executing getStatus: git --no-optional-locks status --untracked-files=all --branch --porcelain=2 -z (took 1.333s)
2022-06-18T21:12:00.749Z - info: [ui] Executing getStatus: git --no-optional-locks status --untracked-files=all --branch --porcelain=2 -z (took 1.344s)
2022-06-18T21:12:36.126Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: .config - xandernp (has token)
